### ISSUE TITLE
Test format failure with LZ4 compression

### ISSUE THREAD

**Issue by agorrod** [wiredtiger/wiredtiger#1784]
Happened in Jenkins here: http://build.wiredtiger.com:8080/job/wiredtiger-test-format-stress/6400/console

Reproduces easily with the following configuration:

```
############################################
#  RUN PARAMETERS
############################################
abort=0
auto_throttle=1
firstfit=0
bitcnt=6
bloom=1
bloom_bit_count=14
bloom_hash_count=11
bloom_oldest=0
cache=30
checkpoints=1
checksum=off
chunk_size=1
compaction=0
compression=lz4
data_extend=0
data_source=lsm
delete_pct=17
dictionary=0
evict_max=4
file_type=row-store
backups=0
huffman_key=0
huffman_value=0
insert_pct=70
internal_key_truncation=1
internal_page_max=9
isolation=read-committed
key_gap=10
key_max=39
key_min=21
leak_memory=0
leaf_page_max=17
logging=0
logging_archive=0
logging_prealloc=1
lsm_worker_threads=4
merge_max=6
mmap=0
ops=100000
prefix_compression=1
prefix_compression_min=6
repeat_data_pct=65
reverse=0
rows=100000
runs=100
split_pct=46
statistics=1
statistics_server=1
threads=24
timer=20
value_max=2622
value_min=6
wiredtiger_config=
write_pct=9
############################################
```

**Comment by agorrod** [comment]
Fix for this one is in:
5fd58d134306470b0177f1ed2bda18b22285b3cb

I'm going to run test/format with lz4 for a while before enabling it for Jenkins again.

**Comment by keithbostic** [comment]
Closed with the merge of 5fd58d1.
